docs/concepts/what-is-an-activity.md

```diff
@@ -39,12 +44,14 @@ It is possible to implement an Activity fully asynchronously by completing it fr
 
 Temporal does not impose any system limit on Activity duration. It is up to the application to choose the timeouts for its execution.
 
-Activities are dispatched to workers through task queues.
-Task queues are queues that workers listen on.
-Task queues are highly dynamic and lightweight.
-They don't need to be explicitly registered. And it is okay to have one task queue per worker process. It is normal to have more than one Activity type to be invoked through a single task queue. And it is normal in some cases (like host routing) to invoke the same Activity type on multiple task queues.
+Activities are dispatched to Workers through Task Queues.
+Task Queues are queues that Workers listen on.
```

Comment:
We explained this point about Activities, Workers, and Task Queues before this, so please remove.